dima/react-movies/src/components/moviePresent/moviePresent.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from "react";
+import { Component } from "react";
```

Comment:
**optional comment**
usually linters give you an error. 
Better to import like this  `import React, { Component } from "react";`

Reply:
fixed